metrics/Gauge.md

```diff
@@ -0,0 +1,80 @@
+# Gauge API Overview
+A `Gauge` is used to record already aggregated metrics, or metrics that can go up and down. Typical examples of gauges would be the number of jobs/entries in a queue or number of threads in a running state.
```

Comment:
already aggregated metrics that can go up and down. we want to add cumulative int/double as well for already aggregated cumulative metrics like total_cpu_usage